GOVERNANCE.md

```diff
@@ -0,0 +1,124 @@
+[high level overview document](README.md).
+
+## Governance
+
+### Governance Goals
+
+The goal of governance for the Diamond Bluff project is to maintain and
+emphasize a technical meritocracy of contributors. Those who contribute the
+most and the best technical solutions, while supporting all contirbutors, have
+the most influence in the technical direction of the project. At the same
+time, a [Board of Directors](#Board of Directors) is established with it's
+own set of responsibilities, to help manage the Diamond Bluff project.
+
+## Membership
+
+The Diamond Bluff project is composed of Platinum, Silver, and Participating
+Academic members. All Platinum and Silver members must be current corporate
+members of the hosting foundation (e.g. Linux Foundation or Open Infrastructure
+Foundation) at any level to participate in the Diamond Bluff project.
+
+Platinum, Silver and Participating Academic Members shall be entitled to:
+
+* Participate in the Diamond Bluff Project meetings, initiatives, events and
+  any other activities.
+* Identify their company or academic institution as a member or participant in
+  the Diamond Bluff project.
+* Platinum members shall be entitled to appoint a representative to the
+  Board of Directors of the Diamond Bluff project, as well as any future
+  committees established by the Board of Directors.
+
+## Structure
+
+At a high level, the Diamond Bluff project is governed by two main bodies which
+work together to achieve the project's goals:
+
+* A Board of Directors
+* A Technical Steering Committee
+
+### Board of Directors
+
+The Diamond Bluff project Board of Directors shall be composed as follows:
+
+* One representative from each Platinum member.
+* One Silver member representative elected annually by the Silver members for
+  every 5 Silver members, with at least one representative, and no more than 3.
+
+The Board of Directors shall have the following responsibilities:
+
+* Approve a budget directing the use of funds raised from all sources of
+  revenue.
+* Elect a Chair of the Diamond Bluff project to preside over meetings,
+  authorize expenditures approved by the budget, and manage any day-to-day
+  operations, and vote on decisions or matters before the Board of Directors.
+
+The Board of Directors shall::
+
+* Demonstrate plans and the means to coordinate with the open source project’s
+  developer community, including on topics such as branding, logos, and other
+  collateral that will represent the community.
+* Engage in a professional manner consistent with maintaining a cohesive
+  community, while also maintaining the goodwill and esteem of the project's
+  foundation, in the open source software community.
+  community.
+* Respect the rights of all trademark owners, including any branding and usage
+  guidelines.
+* Engage the project's foundation for all press and analyst relations activities.
+* Upon request, provide information regarding project participation, including
```

Comment:
Not sure why this needs to be in there?  Won't the foundation have more info about events than we will?

Reply:
Superficial, yes. But a deeper analysis, likely not as much as the maintainers and participants.